ts/codec-template.ts.j2

```diff
@@ -110,7 +110,12 @@ const RESPONSE_{{ to_upper_snake_case(param.name) }}_OFFSET = {% if loop.first %
 const EVENT_{{ to_upper_snake_case(event.name)}}_{{to_upper_snake_case(param.name)}}_OFFSET = {% if loop.first %}PARTITION_ID_OFFSET + BitsUtil.INT_SIZE_IN_BYTES{% else %}EVENT_{{ to_upper_snake_case(event.name)}}_{{ to_upper_snake_case(loop.previtem.name)}}_OFFSET + BitsUtil.{{loop.previtem.type.upper()}}_SIZE_IN_BYTES{% endif %};
     {% endfor %}
 {% endfor %}
-{% if method.response.params|length > 0 %}
+{#RESPONSE PARAMETERS#}
+{% set noResponseValue = method.response.params|length == 0 and response_new_params|length == 0 %}
```

Comment:
Minor: `{% set noResponseValue = method.response.params|length == 0 %}` should be enough here since response_new_params should be of length 0 when there is no response params

Reply:
TBH I was looking at `codec-template.java.j2` when I was doing this change. But it makes sense. Simplified templates for both Node.js and Java in 09e9223a5fc039a34185823880674bc17f2b72a5